content/en/docs/guides/monitoring.md

```diff
@@ -70,6 +70,14 @@ You can access Grafana using port forwarding:
 kubectl -n monitoring port-forward svc/kube-prometheus-stack-grafana 3000:80
 ```
 
+## Log into the Grafana dashbooard
+To log into the Grafana dashboard, use the following credentials:
+```
+username: admin
+password: prom-operator
+```
+These are the default values from the [kube-prometheus helm chart](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L620)
```

Comment:
```suggestion
## Log in to the Grafana dashbooard

To log in to the Grafana dashboard, you can use the default credentials from the [kube-prometheus helm chart](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L620):

```yaml
username: admin
password: prom-operator
```
```

Reply:
* "Log into" -> "log in to"
* Explain where the credentials are from in a single sentence
* Add `yaml` to example block for highlighting